tool/update_version.dart

```diff
@@ -284,43 +293,43 @@ class AutoUpdateCommand extends Command {
   AutoUpdateCommand() {
     argParser.addOption('type',
         abbr: 't',
-        allowed: ['dev', 'dev,patch', 'dev,major', 'patch', 'minor', 'major'],
+        allowed: ['release', 'dev', 'patch', 'minor', 'major'],
         allowedHelp: {
+          'release': 'strips any pre-release versions from the version.',
           'dev':
-              'bumps the version to the next dev pre-release value (minor by default)',
-          'dev,patch': 'bumps the version to the next dev pre-patch value',
-          'dev,major': 'bumps the version to the next dev pre-major value',
-          'patch': 'bumps the version to the next patch value',
-          'minor': 'bumps the version to the next minor value',
-          'major': 'bumps the version to the next major value',
+              'bumps the version to the next dev pre-release value (minor by default).',
+          'patch':
+              'bumps the version to the next patch value, and sets the dev version to 0.',
+          'minor':
+              'bumps the version to the next minor value, and sets the dev version to 0.',
+          'major':
+              'bumps the version to the next major value, and sets the dev version to 0.',
```

Comment:
it would be helpful if each of these help texts had an example like 2.20.0 --> 2.21.0